napari/_tests/test_view_layers.py

```diff
@@ -159,6 +159,6 @@ def test_kwargs_passed(monkeypatch):
         scale=(1, 2, 3),
     )
     assert viewer_mock.mock_calls == [
-        call(title='my viewer', ndisplay=3),
+        call(title='my viewer'),
```

Comment:
I had to change this test since `ndisplay` is now only changed after the call to `viewer.open`. I would've thought based on the documentation of `MagicMock` that I could simply assert at the end that `viewer_mock.dims.ndisplay == 3`, but I get an `AttributeError` when trying to access `Dims`. @tlambert03 maybe you can help me - I thought the point of passing the `Viewer` as a spec to `MagicMock` is that you'd get to access attributes and attributes of attributes the same way as you usually would?